src/editor/tools/info_tool.cc

```diff
@@ -224,5 +224,7 @@ int32_t EditorInfoTool::handle_click_impl(const Widelands::NodeAndTriangle<>& ce
 
 	multiline_textarea->set_text(as_richtext(buf));
 
+	w->initialization_complete();
```

Comment:
Add a TODO(..) that this should be come a subclass of UI::Window() as this is a kind of misusage.